extensions-core/parquet-extensions/src/test/java/org/apache/druid/data/input/parquet/CompatParquetReaderTest.java

```diff
@@ -305,7 +314,12 @@ required group nestedIntsColumn (LIST) {
                                 + "    } ]\n"
                                 + "  }\n"
                                 + "}";
-    Assert.assertEquals(expectedJson, DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues()));
+    try {
+      JSONAssert.assertEquals(expectedJson, DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues()), false);
+    } 
+    catch (JSONException jse) {
```

Comment:
What is the reason for adding try-catch blocks? I see that you are any way throwing RuntimeException in catch block, what is the use of changing the type of exception